wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md

```diff
@@ -0,0 +1,24 @@
+---
+tags:
+  - medals
+  - medal hunting
+  - gameplay
+---
+
+# osu! Medal Hunters Discord server
+
+**osu! Medal Hunters** is a [Discord](https://discordapp.com) server for medals hunters community made by [Osekai](https://osekai.net) team. As the name states, this server is used for osu! medals hunting, discussion of medals and osu! in general.
+
+The permanent invite link is <https://discord.gg/osumedalhunters>.
+
+## Verification
+
+New members can send messages on server by default, but in order to be able to send images and have medals percentage next to their nickname they have to authenticate using authentication link sent by server private bot, Osekai Authenticator!
+
+![Authentication Message Example](img/authentication_example.png "An example of bot authentication message when you join server") ![Medals Percentage Example](img/medals_percentage.png "An example of medals percentage next to user nickname")
+
+Members can't change their server nickname, so this percentage is showing their real medals completion progress!
+
+## Server purpose
+
+osu! Medal Hunters server is mostly about osu! medals, so this is the place where all the medal solving magic happens! A lot of medal hunters work together in order to find a solution for each new Hush-Hush medal added into the game. Every time new medals release, it's announced by one of Osekai staff members, threads for each Hush-Hush medal are created and the hunt begins!
```

Comment:
I guess I can live with this many exclamations, but doesn't this read as a repetition of "it's about medals, and medal hunting" over and over, in different ways?